lib/nexpose/vulnerability.rb

```diff
@@ -112,12 +112,15 @@ def add_bc_to_ssl_cipher_list(source)
     def cleanup_html(source)
       result = source.to_s
       result.gsub!(/<ContainerBlockElement>(.*?)<\/ContainerBlockElement>/m){|m| "#{ $1 }"}
-      result.gsub!(/<Paragraph preformat=\"true\">(.*?)<\/Paragraph>/m){|m| "\nbc. #{ $1 }\n\n"}
+      result.gsub!(/<Paragraph preformat=\"true\">\s*(.*?)<\/Paragraph>/im){|m| "\nbc. #{ $1 }\n\n"}
```

Comment:
Is this something we would also want for the ContainerBlockElement? 

We could also do `$1.strip` if we just want to take whitespace off either side of the capture. The only difference between the two styles is one doesn't add more to the regex 😄 but I'm impartial to either method.

Reply:
I don't think we need this for ContainerBlockElement as it's top-level in the XML sample that we have so wouldn't have spaces to strip. But, I ended up needing to apply it to a bunch of other Paragraph and List tags to get things behaving.